tests/kafkatest/services/mirror_maker.py

```diff
@@ -133,12 +145,19 @@ def start_node(self, node):
         node.account.ssh("mkdir -p %s" % MirrorMaker.LOG_DIR, allow_fail=False)
 
         # Create, upload one consumer config file for source cluster
-        consumer_props = self.render('consumer.properties', zookeeper_connect=self.source.zk.connect_setting())
+        if self.new_consumer:
+            consumer_props = self.render("new_consumer.properties", bootstrap_servers=self.source.bootstrap_servers())
```

Comment:
Not really a big deal, but almost all of this could just be made logic in a single template. I suppose the preference for where this type of logic lives is personal taste, but I like isolating a lot of it to the template so that the service class can focus on *how* to run the service and read more cleanly without having to try to scan through a bunch of conditionals used for adjusting service parameters. In fact, I'm wondering if it makes sense to even convert some of the command line construction into templates since it's pretty messy in the python code but almost all just basic substitutions...

In addition, note that you shouldn't need the extra parameters to `render` here. You can invoke those methods directly in the template. I think we're having to add these parameters because we're reusing templates across multiple services (consumer performance, MM)? Even though they are both configuring consumers, it might make sense to have separate template since sharing the template makes it easy to accidentally break tests that you're not working on (e.g. any non-MM test using console consumer) by modifying the template for an entirely different test (the MM test).